crates/turbopack-ecmascript/tests/analyzer/graph/free-vars/resolved-effects.snapshot

```diff
@@ -0,0 +1,3 @@
+0 -> 3 member call = ???*0*["from"]("Hello World")
+- *0* FreeVar(Buffer)
+  ⚠️  unknown global
```

Comment:
does this not get detected in this testcase?

Reply:
Indeed... hmm...

Reply:
fixed